app/presenters/content_item/contents_list.rb

```diff
@@ -51,7 +51,7 @@ def first_item_content
       first_item_text = ''
 
       until element.name == 'h2'
-        first_item_text += element.text if element.name == 'p'
+        first_item_text += element.text if element.name.in? %w(p ul ol)
```

Comment:
There's a bit of a performance hit in constructing an array within a loop like this.
Ideally we'd use a constant or at least a variable initialised outside the loop.
Also it might be worth amending a test to cover content in a list.